shared/rakefiles/stackdriver.rb

```diff
@@ -68,6 +68,36 @@ def get_alert_policy_identifier(alert_policy)
   return result
 end
 
+def compare_alert_policies(stackdriver_alert_policy, alert_policy)
+  stackdriver_alert_policy = JSON.parse(stackdriver_alert_policy.to_hash.to_json)
+  ["name", "creation_record", "mutated_by", "mutation_record"]. each do |attribute|
```

Comment:
I am concerned about how tightly coupled this is to the (apparently rapidly-evolving) Stackdriver API. I foresee a lot of build failures caused by Stackdriver's JSON format changing periodically.

Do you think an allowlist-based approach (only compare these keys) would be better than a denylist-based approach (compare all but these keys, as you're doing now)? It might provide us a little insulation from API changes.

Other than this,  I'm not really sure what to do. Maybe we should follow Stepan's suggestion and look at moving this functionality back to Terraform in the short-term. Perhaps we should apply this (one last band-aid), see how much it improves things, and then re-evaluate?

Reply:
I don't think we're going to have any issues related to Stackdriver's API changes unless we update `google-cloud-monitoring` / `google-cloud-logging` gems.

I opened https://github.com/gpii-ops/exekube/pull/28 to pin them down to their current versions.

> Do you think an allowlist-based approach (only compare these keys) would be better than a denylist-based approach

Don't think this matters, because of previous point.

> Perhaps we should apply this PR (one last band-aid), see how much it improves alerting stability, and then re-evaluate?

Agree with this.